From 6015297ba3df58cbc40e508cfc9cffd9e333b7b7 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Wed, 11 May 2022 17:42:08 +0200
Subject: [PATCH 208/841] feat(stm32mp2-fdts): change name of OTP
 part_number_otp

Change name of OTP part_number_otp, aligned with OP-TEE
and remove unnecessary alias.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 499d7cb2ace6c5b5298eace4ab4a5689d08c066d feat(stm32mp2-fdts): change name of OTP part_number_otp]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I65352f665c9b438a398c015b01bd0325c3854a1c
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/251863
Tested-by: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 fdts/stm32mp251.dtsi            | 2 +-
 plat/st/stm32mp2/stm32mp2_def.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/fdts/stm32mp251.dtsi b/fdts/stm32mp251.dtsi
index 521125937..48cd7bd9b 100644
--- a/fdts/stm32mp251.dtsi
+++ b/fdts/stm32mp251.dtsi
@@ -288,7 +288,7 @@
 			uid_otp@14 {
 				reg = <0x14 0xc>;
 			};
-			part_number_otp: otp9@24 {
+			part_number_otp@24 {
 				reg = <0x24 0x4>;
 			};
 			nand_otp: otp16@40 {
diff --git a/plat/st/stm32mp2/stm32mp2_def.h b/plat/st/stm32mp2/stm32mp2_def.h
index e485e815a..7c1621d6c 100644
--- a/plat/st/stm32mp2/stm32mp2_def.h
+++ b/plat/st/stm32mp2/stm32mp2_def.h
@@ -296,7 +296,7 @@ enum ddr_type {
 #define STM32MP2_UPPER_OTP_START		0x100U
 
 /* OTP labels */
-#define PART_NUMBER_OTP				"otp9"
+#define PART_NUMBER_OTP				"part_number_otp"
 #define PACKAGE_OTP				"package_otp"
 #define HW2_OTP					"hw2_otp"
 #define NAND_OTP				"otp16"
-- 
2.47.2

